src/time_sorting.py

```diff
@@ -0,0 +1,28 @@
+"""Time how long a sorting algorithm takes."""
+import timeit
+
+
+def time_sorting(algo):
+    """Print results when checking the time."""
+    size = 990
+    tries = 10000
+    list_one = list(range(size))
+    time1 = timeit.timeit(lambda: algo(list_one), number=tries)
+    print("""
+        Best case scenario: all integers 0-989 in ascending order.
+        {} algorithm
+        Test runs: {}
+        Total time: {}
+        Average Time: {}
+        """.format(algo.__name__, tries, time1, time1 / tries))
+
+    list_two = list(range(size))
+    list_two.reverse()
+    time1 = timeit.timeit(lambda: algo(list_two), number=tries)
+    print("""
+        Worst case scenario: all integers 0-989 in descending order.
```

Comment:
I like your reuse of this code across your different sorting algorithms. It is nice to keep thing DRY when possible. Nice thinking! However, does this current implementation apply to all your sorting algorithms in the same way? Do all sorting algorithms have the same best and worst case conditions? How might we improve this?